ui/app/templates/components/job-page.hbs

```diff
@@ -28,6 +28,11 @@
         "job-page/parts/job-client-status-summary" job=@job
       )
       Children=(component "job-page/parts/children" job=@job)
+
+      StatusPanel=(component
+        "job-status/panel" job=@job
```

Comment:
`nit` `question`:  You have two very closely named component `job-page/parts/job-status-panel` which is a very bare bones components. And `job-status/panel`. When you read every other invocation in this file you'll notice they all stem from `job-page/parts/**`. Are you doing this intentionally?

Reply:
Yep, I've since-removed components/job-page/parts/job-status-panel in favour of components/job-status/panel. I had wanted to avoid the prospect of nesting components unnecessarily deep (multiple `::`s in templates references, etc.) but I'm not opposed to just putting what's currently in `/panel` under `/job-parts`.

I guess one of the draw backs of doing it that way is there is already a desire to have these panels outside of the context of an individual job (for example, having them on the /jobs/index route). In that case, /job-parts seems like it wouldn't be an accurate place to store them.